util/test/test_dftool.py

```diff
@@ -148,7 +130,8 @@ def test_noarg_commands():
                 output_s += proc.stdout.readline().decode("ascii")
 
         return_code = proc.wait()
-        assert return_code == 0, f"'dftool {argv1}' failed. \n{output_s}"
+        # bad commands - such as querymany - will still return 0 and do not fail
+        assert return_code == 0, print(f"'dftool {argv1}' failed. \n{output_s}")
```

Comment:
This "change" is wrong, because it does not follow proper `assert` syntax. It was fine before, don't change it.

Reply:
I see, thank you.

Reply:
I have kept the comment for prosperity of understanding what's going on inside the subprocess (i.e. we know some tests fail but that's the test), and reverted the print statement.